python/test_file_generator.py

```diff
@@ -948,17 +933,6 @@ def v637_1_file(self):
 		print("writing {name}".format(name=name))
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
-	def v637_2_file(self):
```

Comment:
Is this file not being used by the Platform?

Reply:
Yes, its just v637.

Reply:
No, just a v637 edit.